src/components/markdown-renderer/markdown-extension/legacy-short-codes/replace-legacy-pdf-short-code.ts

```diff
@@ -13,8 +13,6 @@ export const legacyPdfShortCode: MarkdownIt.PluginSimple = (markdownIt) => {
   markdownItRegex(markdownIt, {
     name: 'legacy-pdf-short-code',
     regex: finalRegex,
-    replace: (match: string) => {
-      return `<a target="_blank" rel="noopener noreferrer" href="${match}">${match}</a>`
-    }
+    replace: (match: string) => `<a href="${match}">${match}</a>`
```

Comment:
Any reason why `target="_blank" rel="noopener noreferrer"` was removed? I think it makes sense here, but one could of course also argue that we simply want to replace this with the same that a simple youtube link would have

Reply:
It's not necessary because this is covered by the link node preprocessor that changes external links.

Reply:
kk